interfaces/builtin/serial_port.go

```diff
@@ -41,7 +42,15 @@ func (iface *SerialPortInterface) String() string {
 
 // Pattern to match allowed serial device nodes, path attributes will be
 // compared to this for validity
-var serialAllowedPathPattern = regexp.MustCompile("^/dev/tty[A-Z]{1,3}[0-9]{1,3}$")
+var serialAllowedPathPattern = regexp.MustCompile(`^/dev/tty[A-Z]{1,3}[0-9]{1,3}$`)
+
+// Strings used to build up udev snippet for VID+PID identified devices. The TAG
+// attribute of the udev rule is used to indicate that devices with these
+// parameters should be added to the apps device cgroup
+var udevVidPidFormat = regexp.MustCompile(`^[\da-fA-F]{4}$`)
+var udevHeader = `IMPORT{builtin}="usb_id"`
+var udevEntryPattern = `SUBSYSTEM=="tty", SUBSYSTEMS=="usb", ATTRS{idVendor}=="%s", ATTRS{idProduct}=="%s"`
+var udevEntryTagPattern = `, TAG+="%s"`
```

Comment:
Gustavo asked me to request that people use `const` string (and do a `[]byte(<string>)` when needed) for security backend variables that hold policy. There are plenty of examples in the interfaces where this isn't happening, and they will be cleaned up.